config/config.php

```diff
@@ -83,6 +83,9 @@
 				'versionstring' => $stableVersionString,
 				'downloadurl' => $stable_mac_url . 'Nextcloud-' . $stableVersion . '.pkg',
 				'web' => 'https://nextcloud.com/install/?pk_campaign=clientupdate#install-clients',
```

Comment:
Not directly related, but with our new webpage I think there is no need of "pk_campaign=clientupdate#install-clients", but the pure https://nextcloud.com/install is enough.

Might be worth changing it?

Reply:
Let me do it in another PR…🙈

Reply:
sure
I was too focused on the task to realize